frontend/src/screens/admin-home.tsx

```diff
@@ -18,13 +18,13 @@ export default function AdminHome() {
         <Grid className="admin">
             <TopNavBar css={{ gridArea: 'header', gridColumnStart: 'span 2' }} />
             <Routes>
-                <Route path="/" element={<Navigate to="/admin/banners/" />} />
-                <Route path="/banners/" element={<AdminBanners />} />
-                <Route path="/approve-studies/" element={<ApproveStudies />} />
-                <Route path="/rewards/" element={<AdminRewards />} />
+                <Route path="/" element={<Navigate to="/admin/banners" />} />
+                <Route path="/banners" element={<AdminBanners />} />
+                <Route path="/approve-studies" element={<ApproveStudies />} />
+                <Route path="/rewards" element={<AdminRewards />} />
                 <Route path="/workspaces/:studyId?" element={<AdminWorkspaces />} />
-                <Route path="/impersonate/" element={<Impersonation />} />
-                <Route path="/reports/" element={<AdminReports />} />
+                <Route path="/impersonate" element={<Impersonation />} />
+                <Route path="/reports" element={<AdminReports />} />
```

Comment:
as I recall, if the url lacks a trailing slash, the app will redirect to have a slash.  adding it at the beginning saves a roundtrip.  Can can test on staging to see if you set that there?

Reply:
I logged into staging as an admin and hit both `/banners` and `/banners/` and it seems to have been functionally equivalent 🤔